[2025-01-07T15:02:15.222+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-01-07T15:02:15.385+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: complex_etl_dag.load_data scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-01-07T15:02:15.426+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: complex_etl_dag.load_data scheduled__2024-02-17T00:00:00+00:00 [queued]>
[2025-01-07T15:02:15.428+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-01-07T15:02:15.642+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load_data> on 2024-02-17 00:00:00+00:00
[2025-01-07T15:02:15.654+0000] {standard_task_runner.py:63} INFO - Started process 2112 to run task
[2025-01-07T15:02:15.682+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'complex_etl_dag', 'load_data', 'scheduled__2024-02-17T00:00:00+00:00', '--job-id', '711', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpdfrdwgam']
[2025-01-07T15:02:15.701+0000] {standard_task_runner.py:91} INFO - Job 711: Subtask load_data
[2025-01-07T15:02:15.955+0000] {task_command.py:426} INFO - Running <TaskInstance: complex_etl_dag.load_data scheduled__2024-02-17T00:00:00+00:00 [running]> on host 4ff5b0867c78
[2025-01-07T15:02:16.547+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='complex_etl_dag' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2024-02-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-02-17T00:00:00+00:00'
[2025-01-07T15:02:16.548+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-01-07T15:02:16.585+0000] {logging_mixin.py:188} INFO - Loading data: data_transformed
[2025-01-07T15:02:16.590+0000] {python.py:237} INFO - Done. Returned value was: data_loaded
[2025-01-07T15:02:16.594+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-01-07T15:02:16.692+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=complex_etl_dag, task_id=load_data, run_id=scheduled__2024-02-17T00:00:00+00:00, execution_date=20240217T000000, start_date=20250107T150215, end_date=20250107T150216
[2025-01-07T15:02:16.791+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-01-07T15:02:16.928+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-07T15:02:16.948+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
